include/wx/dcsvg.h

```diff
@@ -16,7 +16,7 @@
 
```

Comment:
All builds, including those without precompiled headers, succeed. It doesn't seem necessary.